### ISSUE TITLE
PA5 defined twice

### ISSUE THREAD

**Issue by dave31418** [juribeparada/MMDVM_HS#129]
I may be reading the code wrong,   but it appears in Arduino.cpp that PA5 is defined both on line 42 as PIN_TXD2 and then again on line 55 as PIN_POCSAG_LED.    I found this just reading the code so I may not be tracing the execution correctly.   But on the surface this doesn't appear correct.

**Comment by juribeparada** [comment]
You are right and that double definition is incorrect. This didn't produce a problem, probably due to most of people are using the "make" command in Linux to build the firmware, or they simply use the binary form. IOArduino.cpp is used when you compile with the Arduino IDE + STM32duino. It will be fixed.
